bpdm-bridge-dummy/src/main/kotlin/com/catenax/bpdm/bridge/dummy/controller/BridgeController.kt

```diff
@@ -19,22 +19,26 @@
 
 package com.catenax.bpdm.bridge.dummy.controller
 
+import com.catenax.bpdm.bridge.dummy.config.BridgeAuthProperties
 import com.catenax.bpdm.bridge.dummy.service.SyncService
 import io.swagger.v3.oas.annotations.Operation
+import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/api/bridge")
 class BridgeController(
-    val syncService: SyncService
+    val syncService: SyncService,
+    val bridgeAuthProperties: BridgeAuthProperties
```

Comment:
Can be removed if a bean for the configuration properties is created. See the comment in the configurationProperties class